### PR TITLE
Integration renaming to PEP8 compatible

### LINKED ISSUE #69: Rename component
# Problem
`ha-google-home` is not [PEP8 compliant name](https://www.python.org/dev/peps/pep-0008/#package-and-module-names) for python module. It should use `_` for spaces.

This is only 1 of 2 custom components with `-` here: https://github.com/home-assistant/wheels-custom-integrations/tree/master/components

All components in home assistant follow this rule: https://github.com/home-assistant/core/tree/dev/homeassistant/components

# Can we rename the whole thing?
I understand it's quite a huge change but it's better to do this earlier.
But this is also an opportunity to choose a better name. Can we use just `google_home` or it's too bold? 🙂
There is no official `google_home` component anymore, so this name should be free.

# What needs to be renamed
- https://github.com/home-assistant/brands/pull/2317
- https://github.com/home-assistant/wheels-custom-integrations/pull/313
- This folder: https://github.com/leikoilja/ha-google-home/tree/master/custom_components/ha-google-home
- Entries in [`manifest.json`](https://github.com/leikoilja/ha-google-home/blob/master/custom_components/ha-google-home/manifest.json)
- Documentation
~~-Repo renaming (GitHub automatically redirects within the same domain to the updated repo name).~~
  ~~- (edit from Ilja) Update link references in @DurgNomis-drol's card [here](https://github.com/DurgNomis-drol/ha-google-home-timers-card)~~
  ~~- (edit from Ilja) Update link references on HA community forum [here](https://community.home-assistant.io/t/solution-to-track-your-google-home-alarms-and-timers-and-trigger-different-home-assistant-events/61534/87)~~

### PR DESCRIPTION
**Pull request by leikoilja**
Renaming from ha-google-home to PEP8 compatible google-home.

Closes #69

### CODE REVIEW

File: custom_components/google_home/manifest.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
   "after_dependencies": ["cloud", "http", "zeroconf"],
-  "codeowners": ["@leikoilja", "@DurgNomis-drol", "@ArnyminerZ"],
+  "codeowners": ["@leikoilja"],
```
Comment (DurgNomis-drol) [inline custom_components/google_home/manifest.json:3]: Not that i have a need to be listed, but why are we removed? 😄

Comment (leikoilja) [reply custom_components/google_home/manifest.json:3]: haha, very fair point. 😄  
Thing is that there are more and more of us joining to contribute (in a meantime let's welcome @KapJI) and i don't know what is the right way of keeping that list up to date. There are multiple places where "codeowners" must be specified, like this and [this](https://github.com/leikoilja/ha-google-home/blob/2bc6b5482b0abcaeaefa7919a4dd7b7d25c9fdcb/pyproject.toml#L5) one. I am happy to add your handles at those places, but what's the right way of doing so? Shall we be adding @KapJI here as well soon? And if so, what is a measurable amount of contribution to add people to the list? 🤔 
You guys have any suggestions?

Comment (KapJI) [reply custom_components/google_home/manifest.json:3]: Guideline by [HA docs](https://developers.home-assistant.io/docs/creating_integration_manifest/#code-owners):
> GitHub usernames or team names of people that are responsible for this integration. You should add at least your  GitHub username here, as well as anyone who helped you to write code that is being included.

I propose adding everyone here who has collaborator access to this repo.

And regarding `pyproject.toml`, we use it only for testing, let's keep only you there.

Comment (DurgNomis-drol) [reply custom_components/google_home/manifest.json:3]: You have some good points 🤔 Let's just leave it until we find a better way to credit/list maintainers.

Comment (leikoilja) [reply custom_components/google_home/manifest.json:3]: Sounds good with me :)

**Comment by leikoilja** [comment]
Also, @DurgNomis-drol do you mind if we make the lovelace cards section a bit more minimalistic like so:
image

**Review by KapJI — APPROVED** [review]
Looks great! 🎉

**Review by DurgNomis-drol — APPROVED** [review]

**Pull request merged**